internal/globaldefault/default.go

```diff
@@ -72,7 +73,8 @@ func SetupDefaultActivation(subshell subshell.SubShell, cfg DefaultConfigurer, r
 		return locale.WrapError(err, "err_globaldefault_rtexes", "Could not retrieve runtime executables")
 	}
 
-	env, err := runtime.Env(false, false)
+	bashifyPaths := rt.GOOS == "windows" && subshell.Shell() != "cmd"
```

Comment:
Could this be a function of the subshell itself? (ie. `subshell.BashifyPaths()`)?

Reply:
I'm not sure about that since it doesn't make sense for the cmd subshell?

Reply:
I would think that a function like that for `cmd` would just return false, but I'll leave this up to you.